src/structures/Inode.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <vector>
+#include "../includes/Inode.hpp"
+
+Inode::Inode(bool src_type):
+            src_type(src_type),
+            number_references(0),
+            sizeof_file(0),
+            last_access_time(time(nullptr)),
+            last_file_modif_time(time(nullptr)),
+            last_inode_modif_time(time(nullptr)),
+            blocks_amount(1) { }
```

Comment:
I think this constructor is useless?? Why do we need inode about which we only know whether it's file or not?